src/components/Error.astro

```diff
@@ -1,15 +1,17 @@
 ---
 import Badge from '../components/Badge.astro';
+import Carat from './icons/Carat.vue';
 
 const { code, message } = Astro.props;
 ---
 
-<li class="not-prose m-0 p-0 flex flex-col py-4 first:pt-0 type-content-primary">
-  <div class="flex flex-row items-center space-x-3">
+<li data-attribute class="group not-prose m-0 p-0 flex flex-col py-4 first:pt-0 type-content-primary">
+  <button class="w-full relative flex flex-row items-center gap-x-3">
+    <Carat class="absolute -left-5 w-3 h-3 text-content-tertiary invisible" id="carat" data-carat />
```

Comment:
One issue with the absolute positioning is that you don't get the cursor-pointer or behaviour of the button as it is outside the bounds of the button.